client/src/App.tsx

```diff
@@ -12,25 +12,29 @@ const App = () => {
   return (
     <Router>
       <Navbar />
-      <Container>
-        <Switch>
-          <Route exact path={Routes.Home}>
-            TODO: Implement home page
-          </Route>
-          <Route path={Routes.Example}>
-            <Example />
-          </Route>
-          <Route path={Routes.AccountEdit}>
-            <AccountCreate />
-          </Route>
-          <Route path={Routes.AccountCreate}>
-            <AccountCreate />
-          </Route>
-          <Route path={Routes.Accounts}>
-            <Accounts />
-          </Route>
-        </Switch>
-      </Container>
+      <Switch>
+        <Route exact path={Routes.Home}>
+          TODO: Implement home page
+        </Route>
+        <Route path={Routes.TransactionsCreate}>
+          <TransactionForm />
+        </Route>
+        <Route path={Routes.TransactionsEdit}>
+          <TransactionForm />
+        </Route>
```

Comment:
Gal vietoj dviejų Route apibrėžimų tam pačiam <TransactionForm /> component padaryti vieną Route su multiple paths?
`<Route path={Routes.TransactionsCreate | Routes.TransactionsEdit}>
    <TransactionForm />
</Route>
`

Reply:
Nežinau ar taip veiktų, galima pabandyt

Reply:
pabandžiau, neveikia, ten turi būt stringas, route string pagal kurį matchina, net jei encapsulatini su string literal per "|", neveikia taip, reikia naudot string array. Something like:
`[Routes.TransactionsCreate | Routes.TransactionsEdit]` would work. Do we want to go there?

Reply:
Manau, kad važiuojam

Reply:
done